CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ INTERNAL:
 
 * provider: Enable request/response logging in `>=DEBUG` mode [GH-153]
 
+IMPROVEMENTS:
```

Comment:
Lets bump the version header to 1.8.0 since we have an improvement

Reply:
sure